tmail-backend/tmail-third-party/openpaas/src/test/java/com/linagora/tmail/DockerOpenPaasPopulateService.java

```diff
@@ -0,0 +1,80 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ *                                                                  *
+ *  This file was taken and adapted from the Apache James project.  *
+ *                                                                  *
+ *  https://james.apache.org                                        *
+ *                                                                  *
+ *  It was originally licensed under the Apache V2 license.         *
+ *                                                                  *
+ *  http://www.apache.org/licenses/LICENSE-2.0                      *
+ ********************************************************************/
+
+package com.linagora.tmail;
+
+import static com.mongodb.client.model.Filters.eq;
+import static com.mongodb.client.model.Updates.push;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import org.bson.Document;
+
+import com.mongodb.reactivestreams.client.MongoClient;
+import com.mongodb.reactivestreams.client.MongoClients;
+import com.mongodb.reactivestreams.client.MongoCollection;
+import com.mongodb.reactivestreams.client.MongoDatabase;
+
+import reactor.core.publisher.Mono;
+
+public class DockerOpenPaasPopulateService {
```

Comment:
The name looks bad. I see nothing connected to docker (we directly connect to mongo).

How about `OpenPaaSProvisioningService` ?